jax/_src/compiler.py

```diff
@@ -35,6 +37,7 @@
 from jax._src.lib import xla_client as xc
 from jax._src.lib import xla_extension_version
 from jax._src.lib.mlir import ir
+from jax._src.lib.mlir.dialects import func, _stablehlo_ops_gen
```

Comment:
Is it possible to get this from the public Python API rather than an `_`-prefixed module?